clang/test/SemaSYCL/intel-fpga-loops.cpp

```diff
@@ -408,44 +374,6 @@ void loop_attrs_compatibility() {
     a[i] = 0;
   [[intel::loop_count(8)]] for (int i = 0; i != 10; ++i)
     a[i] = 0;
-
-  // no diagnostics are expected
-  [[intel::fpga_pipeline]] [[intel::loop_coalesce]] for (int i = 0; i != 10; ++i)
-    a[i] = 0;
-
-  // no diagnostics are expected
-  [[intel::fpga_pipeline]] [[intel::nofusion]] for (int i = 0; i != 10; ++i)
-    a[i] = 0;
-  // no diagnostics are expected
-  [[intel::fpga_pipeline]] [[intel::loop_count_avg(8)]] for (int i = 0; i != 10; ++i)
-    a[i] = 0;
-  [[intel::loop_count_min(8)]]
-  for (int i = 0; i != 10; ++i)
-      a[i] = 0;
-  [[intel::loop_count_max(8)]]
-  for (int i = 0; i != 10; ++i)
-      a[i] = 0;
-  [[intel::loop_count(8)]] for (int i = 0; i != 10; ++i)
-      a[i] = 0;
```

Comment:
Thanks @premanandrao for reviews.

I have removed 419-429 tests because we are testing here all loop_count* attribute with fpga_pipeline. So no longer need this.

There is already a test for testing all loop_count* attribute with disable_pipeline (line 368-376)
```
// no diagnostics are expected
  [[intel::disable_loop_pipelining]] [[intel::loop_count_avg(8)]] for (int i = 0; i != 10; ++i)
    a[i] = 0;
  [[intel::loop_count_min(8)]] for (int i = 0; i != 10; ++i)
    a[i] = 0;
  [[intel::loop_count_max(8)]] for (int i = 0; i != 10; ++i)
    a[i] = 0;
  [[intel::loop_count(8)]] for (int i = 0; i != 10; ++i)
    a[i] = 0;
```

Reply:
Thanks for explaining.